scripts/zsh.pl

```diff
@@ -1,19 +1,25 @@
 #!/usr/bin/env perl
 
-# Generate ZSH completion
+# Generate ZSH or FISH completion
+# Usage: zsh.pl [path to curl] [fish|zsh]
 
 use strict;
 use warnings;
 
 my $curl = $ARGV[0] || 'curl';
+my $shell = $ARGV[1] || 'zsh';  # zsh or fish
```

Comment:
This makes the path to curl possible to omit if running for zsh, but not for fish which makes for poor user experience. If we go down the route of multi-shell support we should go for proper getopts parsing.

Reply:
Agreed, I'll integrate [Getopt::Long](https://perldoc.perl.org/5.26.0/Getopt/Long.html) …

Reply:
Done.